frontend/scripts/react-components/tool/tool-modal/tool-modal.component.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BaseModal from 'react-components/tool/tool-modal/base-modal';
+import SimpleModal from 'react-components/shared/simple-modal/simple-modal.component';
+import 'react-components/tool/tool-modal/tool-modal.scss';
+
+export default function ToolModal({ items, selectedItem, onChange, activeModal, setActiveModal }) {
+  return activeModal ? (
```

Comment:
why not pass this in the `isOpen` prop?

Reply:
We need the activeModal to render the BaseModal and its easier not to render the modal. Do you think they'll be some inconveniences?

Reply:
ok